Data/Localizations/ru.lua

```diff
@@ -223,58 +218,58 @@ L.GDKP_BIDDER_NEW_BID_LABEL = "Новая ставка";
 L.GDKP_BIDDER_STOP_AUTO_BID = "Остановить автоматическую ставку";
 L.GDKP_BIDDER_THEYRE_TOP_BIDDER = "Максимальная ставка: %s от %s";
 L.GDKP_BIDDER_YOURE_TOP_BIDDER = "Максимальная ставка: %s от вас";
-L.GDKP_CREATE = "Создайте новый сеанс GDKP.";
-L.GDKP_CREATE_MANAGEMENT_CUT = "Управленческое сокращение %";
+L.GDKP_CREATE = "Создать сессию GDKP."; 
+L.GDKP_CREATE_MANAGEMENT_CUT = "Доля РЛа%"; 
 L.GDKP_CREATE_SESSION_INVALID_CUT = "Management Cut должно быть пустым или иметь значение от 0 до 99 (без знака %!)";
-L.GDKP_CREATE_SESSION_INVALID_TYPE = "Выберите тип сеанса!";
-L.GDKP_CREATE_SESSION_SWITCH = "Переключиться на этот сеанс";
-L.GDKP_CREATE_SESSION_TYPE_INFO = [[
+L.GDKP_CREATE_SESSION_INVALID_TYPE = "Выберите тип сессии!"; 
+L.GDKP_CREATE_SESSION_SWITCH = "Сделать активной"; 
+L.GDKP_CREATE_SESSION_TYPE_INFO = [[|c00E7BB00Мультиаукцион позволяет вам делать ставки 
```

Comment:
Same here. The color changes could very well be better, but let's try and make sure they're mostly the same across locales :)